jenkins/seeds/runBuildPlan.groovy

```diff
@@ -6,7 +6,7 @@ def dateFormat = new SimpleDateFormat("yyyy-MM-dd")
 def dateString = dateFormat.format(date)
 
 def scalaRepoUrl = "https://github.com/lampepfl/dotty.git"
-scalaVersion = "3.0.0-RC3-bin-COMMUNITY"  // TODO compute version from latest master
+scalaVersion = "3.0.0-RC3-bin-COMMUNITY-SNAPSHOT"  // TODO compute version from latest master
```

Comment:
You'll also need to update the version inside `env.sh`. Otherwise the compiler won't compile while running the demo script test

Reply:
done